Training/python/plot_quantile_ranges.py

```diff
@@ -145,16 +147,18 @@ def main(
 
                 ### check for anomalous behaviour
                 suspicious_dict = {}
-                suspicious_dict['left_within'] = clamp_range_left > two_sigma_left
-                suspicious_dict['right_within'] = clamp_range_right < two_sigma_right
-                # suspicious_dict['left_beyond'] = clamp_range_left < five_sigma_left
-                # suspicious_dict['right_beyond'] = clamp_range_right > five_sigma_right
-                suspicious_dict['one_sigma_empty'] = one_sigma_left == one_sigma_right
-                suspicious_dict['two_sigma_empty'] = two_sigma_left == two_sigma_right
-                suspicious_dict['three_sigma_empty'] = three_sigma_left == three_sigma_right
-                suspicious_dict['five_sigma_empty'] = five_sigma_left == five_sigma_right
+                has_None = has_None or (None in [median, min_value, max_value, one_sigma_left, two_sigma_left, three_sigma_left, five_sigma_left, one_sigma_right, two_sigma_right, three_sigma_right, five_sigma_right])
+                if not has_None:
```

Comment:
So given possible `None` presence this can be changed into:

```python
suspicious_dict = {}
is_suspicious = False
has_None = None in [mean, median, min_value, max_value, one_sigma_left, two_sigma_left, three_sigma_left, five_sigma_left, one_sigma_right, two_sigma_right, three_sigma_right, five_sigma_right])
if not has_None:
    clamp_range_left = mean + var_scaling['lim_min']*var_scaling['std']
    clamp_range_right = mean + var_scaling['lim_max']*var_scaling['std']
    clamp_range = [clamp_range_left, clamp_range_right]
    # suspicious_dict['...'] = ... # all the suspicious definitions as before
    is_suspicious = any(suspicious_dict.values())
elif mean is None:
    print(f'       {var_name}, {cone_type}: mean is empty')
    continue
else:
    print(f'       {var_name}, {cone_type}: quantiles are empty')
    continue   
``` 
Then, the remaining part starting from `if is_suspicious or not only_suspicious:` doesn't require any changes w.r.t. current master branch.